board/model_manager.py

```diff
@@ -10,21 +13,25 @@ def __init__(self, recommender_name, train_sampler, val_sampler, test_sampler, e
         self.test_sampler = test_sampler
         self.evaluation_metric = evaluation_metric
         self.path_to_dataset = path_to_dataset
+        
+        logging.basicConfig(format=FORMAT)
+        self.logger = logging.getLogger('modelmanager')
```

Comment:
We should have memory efficient logging. Need to understand- 
1. Internals of python logger
2. Decide how many unique instances of logger to use for the project (if we need more than one).
Approving for now. Pl create an enhancement issue `Mem efficient logging`

Reply:
Lets use `logging.getLogger(__name__)` for now